planner_cspace_msgs/action/MoveWithTolerance.action

```diff
@@ -2,6 +2,7 @@ geometry_msgs/PoseStamped target_pose
 float64 goal_tolerance_lin
 float64 goal_tolerance_ang         # An angle tolerance to the goal point from the robot.
 float64 goal_tolerance_ang_finish  # An angle tolerance that the robot on the goal point should direct to a certain angle.
+bool is_continuous_movement_mode
```

Comment:
I think `is_*` sounds like a status.
As it's a command, `continuous_movement_mode` or `continuous_movement` would be better.

Reply:
Renamed to continuous_movement_mode.